Lib/ldap/ldapobject.py

```diff
@@ -833,6 +832,11 @@ class ReconnectLDAPObject(SimpleLDAPObject):
   * retry_delay: specifies the time in seconds between reconnect attempts.
 
   This class also implements the pickle protocol.
+
+  .. versionadded:: 4.0
```

Comment:
Any reasons on why this couldn't go into 3.5? Doesn't look to be breaking any plausible use cases?

Reply:
don't know - your decidion. for other changes I was asked to put it into 4.0 and didn't thought that 3.5 is still developed.

Reply:
This is considered a fix, not behaviour change, correct? Then I'd think it qualifies for 3.x, unless @droideck disagrees?

Reply:
I'd say it's a minor behaviour change (we stated previously in the docs that we reconnect only on `ldap.SERVER_DOWN`, but now it's `ldap.SERVER_DOWN,ldap.UNAVAILABLE, ldap.CONNECT_ERROR, ldap.TIMEOUT`)

But I think it's really minor and we can release it as a bugfix. So yep, I fully agree with putting it to 3.5!

Reply:
OK, changed to 3.5